helm/csi-powermax/templates/node.yaml

```diff
@@ -105,11 +105,9 @@ spec:
               value: "{{ .Values.kubeletConfigDir }}/plugins/powermax.emc.dell.com/disks"
             - name: X_CSI_MANAGED_ARRAYS
               value: {{ $arraysStr | replace " " "," | quote }}
-            {{- if ne .Values.csireverseproxy.enabled true }}
             {{- $managementServer := first .Values.global.managementServers }}
```

Comment:
Remove $managementServer, it was usable without proxy.

Reply:
it is used to fetch endpoint below